faq/serverless-jobs.mdx

```diff
@@ -10,9 +10,27 @@ category: serverless
 productIcon: ServerlessJobsProductIcon
 ---
 
-## What are Serverless Jobs?
+## What is serverless computing and how does it differ from traditional cloud hosting?
 
-Scaleway Serverless Jobs is a fully managed service that enables efficient execution of batch computing workloads. It automates management tasks, allowing users to run large-scale batch jobs with ease.
+Serverless computing is a cloud execution model where the cloud provider dynamically manages the allocation of compute resources. Unlike traditional hosting models, you don’t need to provision, scale, or maintain servers. Instead, you focus solely on writing and deploying your code, and the infrastructure scales automatically to meet demand.
+
+## Why consider using Serverless Containers, Functions or Jobs for my projects?
+
+These services allow you to build highly scalable, event-driven, and pay-as-you-go solutions. Serverless Containers and Functions help you create applications and microservices without worrying about server management, while Serverless Jobs let you run large-scale, parallel batch processing tasks efficiently. This can lead to faster development cycles, reduced operational overhead, and cost savings.
```

Comment:
```suggestion
These services allow you to build highly scalable, event-driven, and pay-as-you-go solutions. Serverless Containers and Functions help you create applications and microservices without worrying about server management, while Serverless Jobs lets you run large-scale, parallel batch-processing tasks efficiently. This can lead to faster development cycles, reduced operational overhead, and cost savings.
```